package.json

```diff
@@ -18,11 +18,7 @@
 		"collectCoverageFrom": [
 			"packages/*/**/*.js"
 		],
-		"coveragePathIgnorePatterns": [
-			"<rootDir>/.*/build.*"
-		],
-		"coverageDirectory": "coverage",
-		"setupTestFrameworkScriptFile": "./packages/jest-console/build/index.js"
+		"preset": "./packages/jest-preset-default/jest-preset.json"
```

Comment:
We will have to include `@wordpress/jest-preset-default` in `devDependencies` and replace this line with:

```json
"preset": "@wordpress/jest-preset-default"
```